pundix/chain.json

```diff
@@ -0,0 +1,173 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Pundi X Chain",
+  "chain_id": "PUNDIX",
+  "website": "https://pundix.com",
+  "pretty_name": "Pundi X Chain",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "px",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Thank you for reminding me. I'll do it right away